clients/nimbus-el/Dockerfile

```
@@ -1,36 +1,39 @@
 # Docker container spec for building the master branch of nimbus.
 
-FROM debian:buster-slim AS build
+FROM debian:testing-slim AS build
+
+SHELL ["/bin/bash", "-c"]
+
+RUN apt-get clean && apt update \
+ && apt -y install curl build-essential git-lfs librocksdb-dev
+
+RUN ldd --version
 
 ARG tag=master
 ARG github=status-im/nimbus-eth1
 
-RUN apt-get update \
- && apt-get install -y --fix-missing build-essential make git libpcre3-dev librocksdb-dev \
- && apt-get clean \
- && rm -rf /var/lib/apt/lists/* /tmp/* /var/tmp/*
-
-RUN git clone --recurse-submodules --depth 1 --branch "$tag" https://github.com/$github
+RUN git clone --branch "$tag" https://github.com/$github
 
 WORKDIR /nimbus-eth1
 
 RUN NPROC=$(nproc); make -j${NPROC} NIMFLAGS="--parallelBuild:${NPROC}" V=1 update
 
-RUN NPROC=$(nproc); make -j${NPROC} NIMFLAGS="--parallelBuild:${NPROC} -d:chronicles_colors=none" nimbus && \
-    mv build/nimbus /usr/bin/
+RUN NPROC=$(nproc); make -j${NPROC} NIMFLAGS="--parallelBuild:${NPROC} -d:chronicles_colors=none" nimbus_execution_client && \
+    mv build/nimbus_execution_client /usr/bin/
 
 # --------------------------------- #
 # Starting new image to reduce size #
 # --------------------------------- #
-FROM debian:buster-slim AS deploy
+FROM debian:testing-slim AS deploy
+
+RUN apt-get clean && apt update \
+ && apt -y install build-essential librocksdb-dev jq curl
+RUN apt update && apt -y upgrade
 
-RUN apt-get update \
- && apt-get install -y librocksdb-dev bash curl jq\
- && apt-get clean \
- && rm -rf /var/lib/apt/lists/* /tmp/* /var/tmp/*
+RUN ldd --version ldd
```

```suggestion
RUN ldd --version
```
Same as other comment, will commit this one and merge.